code/game/objects/items/bodybag.dm

```diff
@@ -131,8 +131,7 @@
 		return FALSE //Only humans.
 	if(mob_to_stuff.stat != DEAD) //Only the dead for bodybags.
 		return FALSE
-	var/mob/living/carbon/human/human_to_stuff = mob_to_stuff
-	if(human_to_stuff.is_revivable() && (check_tod(human_to_stuff) || issynth(human_to_stuff)))
```

Comment:
Why remove issynth check? It seems that synths can now be bodybagged.

Reply:
hum true, i misunderstood the check